spec/abstractops.html

```diff
@@ -1035,12 +1035,7 @@ <h1>ISO 8601 grammar</h1>
           UTCDesignator
 
       TimeZoneNumericUTCOffsetNotAmbiguous :
-          `+` TimeZoneUTCOffsetHour
-          U+2212 TimeZoneUTCOffsetHour
-          TimeZoneUTCOffsetSign TimeZoneUTCOffsetHour `:` TimeZoneUTCOffsetMinute
-          TimeZoneUTCOffsetSign TimeZoneUTCOffsetHour TimeZoneUTCOffsetMinute
-          TimeZoneUTCOffsetSign TimeZoneUTCOffsetHour `:` TimeZoneUTCOffsetMinute `:` TimeZoneUTCOffsetSecond TimeZoneUTCOffsetFraction?
-          TimeZoneUTCOffsetSign TimeZoneUTCOffsetHour TimeZoneUTCOffsetMinute TimeZoneUTCOffsetSecond TimeZoneUTCOffsetFraction?
+          TimeZoneUTCOffset but not `-` TimeZoneUTCOffsetHour
```

Comment:
Is this a valid way of expressing this? I don't think I saw any existing examples of "but not" followed by a sequence; only single literals, nonterminals, or "one of" (e.g., "but not one of `\`, `'`, or _LineTerminator_")

Reply:
I believe so based on [Grammar Notation](https://tc39.es/ecma262/multipage/notational-conventions.html#sec-grammar-notation) («_using the phrase “<b>but not</b>” and then indicating the expansions to be excluded_»), but if not then I think it could instead use a negative lookahead like [|ForStatement|](https://tc39.es/ecma262/multipage/ecmascript-language-statements-and-declarations.html#prod-ForStatement):
```suggestion
          [lookahead != `-` TimeZoneUTCOffsetHour] TimeZoneUTCOffset
```